Changes to mesh ingress information (#2748)
ansible/aap-docs: downstream/modules/platform/proc-connecting-nodes-through-mesh-ingress.adoc

    name:
    namespace:
spec:
    deployment_name: 
----
+
Where:

* *apiVersion*: defines the versioned schema of this representation of an object. 
Servers should convert recognized schemas to the latest internal value, and might reject unrecognized values.
+

> Set the value to `automationcontroller.ansible.com/v1alpha1`.

Reviewer: put this on line 27 in the template since its static 

```
apiVersion: automationcontroller.amiable.com/v1alpha1
```

* *kind*: Is the type of node to create. 

> +

Reviewer: kind is static too it should always be `AutomationControllerMeshIngress`

Set the value to `AutomationControllerMeshIngress`. 
`AutomationControllerMeshIngress` controls the deployment and configuration of mesh ingress on {ControllerName}.
* *name*: is the name of the mesh ingress node. You must choose this name TBD

> * *namespace*: is which Kubernetes namespace to deploy the mesh ingress into and is determined by the operator. TBD

Reviewer: this need to be in the same namespace where the AutomationController the MeshIngress is going to be attach to

> * *namespace*: is which Kubernetes namespace to deploy the mesh ingress into and is determined by the operator. TBD

Reviewer: ```suggestion
* *namespace*: is which Kubernetes namespace to deploy the mesh ingress into. TBD
```

"is determined by the operator" I don't understand what that means this is an info the user need to provide.

This must be in the same namespace as the {ControllerName} that the mesh is connected to.

> * *deployment_name*: Use the `deployment_name' `aap-controller`.

Reviewer: ```suggestion
* *deployment_name*: is the AutomationController instance that this mesh ingress would be attached to.
```

Using the following command to determine the `deployment_name`
+
----

> oc get automation_controllers

Reviewer: ```suggestion
oc get automationcontroller
```

----
+

> always supplies `aap-controller`.

Reviewer: i don't think this is correct the user will need to provide which instance of AutomationController do they want to attach the mesh ingress that they are creating

. Apply this YAML file using:
+
----
oc apply -f oc_meshingress.yml
----
+

> This runs the playbook associated with `AutomationControllerMeshIngress`, and creates the hop node with the `name` you supplied.

Reviewer: ```suggestion
Applying the yaml will create the `AutomationControllerMeshIngress` resource and the operator will creates a hop node in controller with the `name` you supplied.
```
